MiniApp/Classes/core/Protocols/MiniAppMessageDelegate.swift

```diff
@@ -197,7 +197,7 @@ public enum MASDKAccessTokenError: Error, MiniAppErrorProtocol {
         case .authorizationFailureError:
             return "AuthorizationFailureError"
         case .error:
```

Comment:
this will be OK on JS Bridge side, yet we are expecting a null key for this case, not empty key